ceph.spec.in

```diff
@@ -505,6 +505,7 @@ Summary:        ceph-mgr rook plugin
 Group:          System/Filesystems
 %endif
 Requires:       ceph-mgr = %{_epoch_prefix}%{version}-%{release}
+Requires:       python-kubernetes
```

Comment:
We will need this to be python3-kubernetes on RHEL 8 and Fedora. And SUSE? @smithfarm

Reply:
@jtlayton shall we put something like
```
python%{_python_buildid}-kubernetes
```
instead ?

Reply:
+1 to `python%{_python_buildid}-kubernetes`

Reply:
I was planning on relying on the "provides" fields for these packages. The python2 package provides python-kubernetes:
```
# rpm -q --provides python2-kubernetes
python-kubernetes = 8.0.0-8.el7
python2-kubernetes = 8.0.0-8.el7
```
But it looks like this may be missing from the python3 package (at least in Fedora). I'll change the patch to include the _python_buildid (and file a bug against the python3 package).

Reply:
@jtlayton I don't think that "missing provides" in the python3 RPM is a bug. AFAIK in both SUSE and RHEL 8, "python" is now a synonym for "python2" (hence the "provides" in the python-kubernetes RPM) but "python3" must always be explicit.

Reply:
Agreed, I don't think it's a bug.